vendor.conf

```diff
@@ -31,21 +30,17 @@ github.com/prometheus/procfs cb4147076ac75738c9a7d279075a253c0cc5acbd
 github.com/Shopify/logrus-bugsnag 577dee27f20dd8f1a529f82210094af593be12bd
 github.com/spf13/cobra 312092086bed4968099259622145a0c9ae280064
 github.com/spf13/pflag 5644820622454e71517561946e3d94b9f9db6842
-github.com/xenolf/lego a9d8cec0e6563575e5868a005359ac97911b5985
 github.com/yvasiyarov/go-metrics 57bccd1ccd43f94bb17fdd8bf3007059b802f85e
 github.com/yvasiyarov/gorelic a9bba5b9ab508a086f9a12b8c51fab68478e2128
 github.com/yvasiyarov/newrelic_platform_go b21fdbd4370f3717f3bbd2bf41c223bc273068e6
-golang.org/x/crypto c10c31b5e94b6f7a0283272dc2bb27163dcea24b
+golang.org/x/crypto e84da0312774c21d64ee2317962ef669b27ffb41
```

Comment:
Why this version? The [commit](https://github.com/golang/crypto/commit/e84da0312774c21d64ee2317962ef669b27ffb41) was on Oct 2018. Should it be more recent one? Also there are no versions in that repo. Not sure what is best version to pick.

Reply:
I did it for parity with the Go version (1.11.x) used by travis to build the distribution project.

Reply:
Sounds good. We should update travis to 1.12 but thats a different PR.